src/city/crud.py

```diff
@@ -0,0 +1,32 @@
+from sqlalchemy import select, insert
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from src.city.models import DBCity
+from src.city.schemas import CreateCity
+
+
+async def get_all_cities(db: AsyncSession):
+    query = select(DBCity)
+    result = await db.execute(query)
+    return [city[0]  for city in result.fetchall()]
+
+
+async def get_city_by_name(db: AsyncSession, city_name: str) -> DBCity:
+    query = select(DBCity).where(DBCity.name == city_name)
+    result = await db.execute(query)
+    city = result.fetchall()
+    if city:
+        return city[0][0]
```

Comment:
The return statement `return city[0][0]` assumes that `city` is a list of tuples with each tuple containing a single element. Ensure this structure is consistent with the query results.